rootfs/docker-entrypoint-initdb.d/003_restore_from_backup.sh

```diff
@@ -20,7 +20,7 @@ gosu postgres pg_ctl -D "$PGDATA" -w restart
 if [[ $(envdir "$WALE_ENVDIR" wal-e --terse backup-list | wc -l) -gt "1" ]]; then
   echo "Found backups. Restoring from backup..."
   gosu postgres pg_ctl -D "$PGDATA" -w stop
-  rm -rf "$PGDATA"
+  rm -rf "$PGDATA/*"
```

Comment:
I think the new comman will keep the `$PGDATA` root folder while deleting every folder and file in it. Is this the intended behavior here?

Reply:
Yes, under the low version of docker, it is possible that $PGDATA will be occupied, like: 

`rm: cannot remove '/var/lib/postgresql/data': Device or resource busy`

Reply:
Understood. This makes sense and is the correct way forward then! :+1: